owncloud/ios-app
# Last item was overlapped by Tab bar

## Description
The last item in file list was overlapped by the tab bar

## Related Issue
#287 

## Motivation and Context
User could not touch and see last item

## How Has This Been Tested?
- open file list
- you need a file list, which more items can be displayed on the screen
- scroll down to the last item
- last item should be completely displayed and touchable 

## Types of changes


## Checklist:
